.devcontainer/gw-container/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM continuumio/anaconda3
+
+RUN conda install -c conda-forge bilby gwpy python-lalsimulation pymultinest
```

Comment:
We may want to use a .yml file to configure the environments, what do you think?

Also, I think the [`gw_requirements.txt`](https://github.com/bilby-dev/bilby/blob/main/gw_requirements.txt) file includes the minimum dependencies we need for the GW dockerfile, though we may wish to add more, e.g. `pytest` and the other dependencies we use for testing.

Reply:
Seems a good solution, i will have a look over this today at afternoon.

Thanks about the context with gw_requirements, i will take that on mind, i will try to provide you a easy solution for add and modify dependencies in case it's necessary.